spec/stack.js

```diff
@@ -18,19 +18,52 @@ describe('Stack', () => {
       expect(() => aStack.push('foo'))
         .to.alter(() => aStack.length(), { from: 0, to: 1 })
     })
+    it('pushes an element to the top of the stack.', () => {
+      const aStack = new Stack()
+
+      aStack.push('bar')
+      aStack.push('foo')
+
```

Comment:
The description should be changed to explain that the element is added to the end of the array
A test could also be added to calculate the length of the array so that it verifies that the latest element was pushed to the bottom